lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -46,18 +46,16 @@ class LighthouseViewerReport {
 
   addListeners() {
     const printButton = document.querySelector('.js-print');
-    if (!printButton) {
-      return;
+    if (printButton) {
```

Comment:
Before, everything was a noop if the print button didn't exist. I changed it to attach listeners based on each button.